test/container/Dockerfile-OpenSuse-latest

```diff
@@ -13,7 +13,7 @@ RUN dnf -y install --setopt=install_weak_deps=False \
     strace libkmod-devel gcc bzip2 xz tar wget rpm-build make git bash-completion \
     sudo kernel dhcp-client qemu-kvm /usr/bin/qemu-system-$(uname -m) e2fsprogs \
     tcpdump iproute iputils kbd NetworkManager btrfsprogs tgt dbus-broker \
-    iscsiuio open-iscsi which ShellCheck procps pigz parted squashfs ntfsprogs \
+    iscsiuio open-iscsi which ShellCheck procps pigz parted squashfs ntfsprogs wicked \
```

Comment:
I don't know if we should spend time and resources in testing the wicked module... I doubt anyone is using it, the dracut module is incomplete (see #1670), and wicked is being phased out from openSUSE (see https://news.opensuse.org/2022/01/28/bash-systemd-libvirt-update-in-tw/).

Reply:
I don't know if it's reason enough to drop it, I don't know how many users might be using it, but at least it's reason enough not to waste time testing it upstream.